test/util/populate/certs.go

```diff
@@ -0,0 +1,25 @@
+package populate
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+)
+
+var dummyPrivateKey *rsa.PrivateKey
```

Comment:
@jim-minter do you have a similar trick for a `*x509.Certificate`. Right now, we cannot unmarshall `Certificates` which were marshalled from the zero-value.

Reply:
I don't immediately have a similar trick - where are you unmarshalling and why?

Reply:
We need to unmarshall a curated yaml into corresponding api structs (internal/external) and then compare those unmarshalled structs with structs which are populated using `populate.Walk`. The converse operation, marshalling a struct populated with `populate.Walk` and comparing the yaml  to a golden yaml, works.

Reply:
This is exactly where the *x509.Certificate* unmarshalling is needed. That test is currently skipped: https://github.com/openshift/openshift-azure/pull/699/files#diff-de120f39e8d8448b9aa430e66c997a97R383